components/mixins/experience-event/events/filloutform.schema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/filloutform",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Fill Out Form",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "web.fillOutForm",
```

Comment:
Should this be web.webpagedetails.fillOutForm as described on wiki? Otherwise, please keep wiki up to date with a revision.

Reply:
we discussed about having a separate section under web fillOutForm as by definition this one does not fit in webPageDetails. Please let me know if you think otherwise